components/browser/state/src/main/java/mozilla/components/browser/state/state/CustomTabSessionState.kt

```diff
@@ -24,7 +25,8 @@ data class CustomTabSessionState(
     val config: CustomTabConfig,
     override val engineState: EngineState = EngineState(),
     override val extensionState: Map<String, WebExtensionState> = emptyMap(),
-    override val contextId: String? = null
+    override val contextId: String? = null,
+    override var lastAccess: Long = System.currentTimeMillis()
```

Comment:
🤔  Not sure if we should have a default value? I don't know if creating that tab would count as accessing it.

Reply:
yes, I will remove it

Reply:
@Vishwa-Mozilla I checked with @eoger who recommended keeping this to `System.currentTimeMillis()` instead of defaulting to `0L`.

We could sync the tabs before accessing them and that might cause issues for the other synced devices.

Reply:
Actually, sorry for going back on my word and for the resulting confusion, let's go with `0L`. I'm concerned this might interact badly with session restore, and it agrees with [the current implementation](https://searchfox.org/mozilla-central/rev/0e09b9191c02097034e46b193930f91c45b7885d/services/sync/modules/engines/tabs.js#209).